armi/settings/tests/test_settings.py

```diff
@@ -41,7 +41,7 @@
 THIS_DIR = os.path.dirname(__file__)
 
 
-class DummyPlugin1(plugins.ArmiPlugin):
+class DummyParamPlugin1(plugins.ArmiPlugin):
```

Comment:
Since the associated req stipulates adding "multiple settings", it might be a good idea to add a second new setting from within this class and test that.

Reply:
Sure, easy. I added a second setting to this class: "avocados".

And added a couple checks to `test_pluginSettings()` below that also verify that registered correctly.